scripts/deploy.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+#bin/sh
+
+echo "Deploy a Non-Signed Staging Release"
+mvn deploy -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -B -DskipTests --settings .travis.settings.xml

```

Comment:
I guess you want to suppress the log, I suggest `mvn -q`

Reply:
Thanks for your reminder, the property `org.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener` 
is only used to get rid of the "Downloading/Downloaded" lines, I think the other logs needs to be retained to view the execution track

Reply:
Ok, we will keep it that way